config.py

```diff
@@ -231,22 +231,22 @@ def _setup_discord_guild_id(cls) -> None:
 
     @classmethod
     def _setup_group_full_name(cls) -> None:
-        raw_group_full_name: str | None = os.getenv("GROUP_NAME")
+        raw_group_name: str | None = os.getenv("GROUP_NAME")
 
-        if raw_group_full_name is not None:
-            raw_group_full_name = raw_group_full_name.strip()
+        if raw_group_name is not None:
+            raw_group_name = raw_group_name.strip()
 
-        if not raw_group_full_name:
+        if not raw_group_name:
             cls._settings["_GROUP_FULL_NAME"] = None
             return
 
-        if not re.fullmatch(r"\A[A-Za-z0-9 '&!?:,.#%\"-]+\Z", raw_group_full_name):
+        if not re.fullmatch(r"\A[A-Za-z0-9 '&!?:,.#%\"-]+\Z", raw_group_name):
             INVALID_GROUP_FULL_NAME: Final[str] = (
```

Comment:
```suggestion
            INVALID_GROUP_NAME: Final[str] = (
```
This message refers to the invalid environment variable, and should have its name updated to reflect that.